### PR TITLE
Expose ExpressionSizeCodePointLimit as env option.

### PR DESCRIPTION
**Pull request by notyourusualaccountname2**
Currently, the parser options cannot be customized when creating a new environment. With this change, one can override the max. number of code points the parser accepts via an env option. This is analogous to ParserRecursionLimit.

### CODE REVIEW

File: cel/options.go [hunk 1]
```diff
@@ -568,3 +568,13 @@ func ParserRecursionLimit(limit int) EnvOption {
 		return e, nil
 	}
 }
+
+// ParserExpressionSizeLimit adjusts the max. number of code points the expression parser is allowed
```
Comment (TristonianJones) [inline cel/options.go]: nit: remove 'max.' and keep the first sentence on a single line.

Thanks!

Comment (notyourusualaccountname2) [reply cel/options.go]: Done.

**Review by TristonianJones — APPROVED** [review]

**Pull request merged**